index.html

```diff
@@ -1,62 +1,199 @@
-          <a href="#" class="menu"><img src="./Enabled.png" alt="Menu button"/></a>
-        </nav>
-      </header>
-      <section class="headline">
-        <h1 class="primary-text">Hey There. I'm Festus</h1>
-        <h2 class="supporting-text">I am a Software Developer</h2>
-        <p class="description">
-          I can help you build a product , feature or website Look through some
-          of my work and experience! If you like what you see and have a project
-          you need coded, don't hestiate to contact me.
-        </p>
-        <ul class="social-media">
-          <li>
-            <a href="#" class="git"
-              ><img src="./Github.png" alt="Github logo"
-            /></a>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./style.css" />
+  <title>Personal Portfolio</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Odibee+Sans&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+</head>
+
+<body>
+
+  <main>
+    <header class="container">
+      <a href="#" class="logo"><img src="./img/Logo_placeholder.png" alt="Logo" /></a>
+      <nav>
+        <a href="#" class="menu"><img src="./img/Enabled.png" alt="Menu button" /></a>
+      </nav>
+    </header>
+    <section class="headline">
+      <h1 class="primary-text">Hey There. I'm Festus</h1>
+      <h2 class="supporting-text">I am a Software Developer</h2>
+      <p class="description">
+        I can help you build a product , feature or website Look through some
+        of my work and experience! If you like what you see and have a project
+        you need coded, don't hestiate to contact me.
+      </p>
+      <ul class="social-media">
+        <li>
+          <a href="#" class="git"><img src="./img/Github.png" alt="Github logo" /></a>
+        </li>
+        <li>
+          <a href="#" class="linkedin"><img src="./img/linkedIn.png" alt="Linkedin logo" /></a>
+        </li>
+        <li>
+          <a href="#" class="angelist"><img src="./img/angelist.png" alt="Angelist logo" /></a>
+        </li>
+        <li>
+          <a href="#" class="twitter"><img src="./img/twitter.png" alt="Twitter logo" /></a>
+        </li>
+        <li>
+          <a href="#" class="vector"><img src="./img/vector.png" alt="Vector logo" /></a>
+        </li>
+      </ul>
+    </section>
+
+      <section class="works-section">
+        <div class="topic-wrapper">
+          <h2 class="topic">My Recent Works</h2>
+          <img src="./img/underline.png" alt="underline" class="underline" />
+        </div>
+        <div class="title-container">
+          <img src="./img/Img_laceholder.png" alt="featured image" class="placeholder" />
```

Comment:
Great job @Enning94 🤝

- [ ] This image is not displaying well. Please try to fix it
![image](https://user-images.githubusercontent.com/63917725/176987810-5573074e-e620-4995-9c3a-0d390f5fb10a.PNG)